### DISCUSSION TITLE
Add an option or variable for automatically peer listening port "Open" state

### DISCUSSION THREAD

**Discussion by ghost** [haugene/docker-transmission-openvpn#2786] [Ideas]
Hi!

My Transmission works great with my VPN.

The problem is that the peer listening port after a reconnection or restart of Transmission often goes to "Closed" instead of "Open" and the sharing speed becomes much worse than when the peer listening port is "Open" .

The state between "Open" and "Closed" is completely random.

Would it be possible to add an option or variable so that Transmission restarts automatically until the peer listening port returns to "Open" state?

![transmission_open_closed](https://github.com/haugene/docker-transmission-openvpn/assets/90903325/9ccb65f6-69f2-4829-a29d-59bde69a9c50)

**Comment by pkishino** [comment]
Please bring this to the transmission board, we don’t have anything to do with transmission functionality
